sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -93,20 +93,11 @@
       <version>1.0.4</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-test-watcher</artifactId>
+      <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-test-watcher;dependency} -->
+      <scope>test</scope>
+    </dependency>
   </dependencies>
-
```

Comment:
Could you port the surefire extension change to check the display in both console and report?

Reply:
Is this what you were looking for in the console output?

![image](https://user-images.githubusercontent.com/48699787/79614717-f66d1000-80b5-11ea-87fa-112ef70823fe.png)

Reply:
IDE is working always. Need to check the surefire report under target.
![image](https://user-images.githubusercontent.com/48036328/79614878-5663b680-80b6-11ea-8ceb-a889324e0888.png)
![image](https://user-images.githubusercontent.com/48036328/79614898-5fed1e80-80b6-11ea-90b9-78487afab06f.png)

Reply:
This still has `surefire-reports` in target.